resources_pages/install_ds_stack_windows.md

```diff
@@ -3,290 +3,540 @@ layout: page
+
+To use Rtools from R in terminal, edit `~/.Rprofile`:
+
+```
+code ~/.Rprofile
+```
+
+and append the following:
+
+```
+# Enable Rtools from R in the terminal
+Sys.setenv(PATH = paste("C:/rtools40/usr/bin/", Sys.getenv("PATH"), sep=";"))
+```
+
+### IR kernel
+
+The `IRkernel` package is needed to make R work in Jupyter notebooks. To enable this kernel in the notebooks, open R *from a terminal* and run the setup via the following two commands:
+
+```
+install.packages('IRkernel')
+IRkernel::installspec()
+```
+
+When asked to select a mirror, pick one at a location close to where you live for faster downloads.
+
+> Note that you cannot use RStudio for this step because it will not be able to find the jupyter installation. R from terminal will since the correct PATH for jupyter is set when the terminal is launched.
+
+To see if you were successful, try running Jupyter Lab and seeing if you have working R kernel. To launch the Jupyter Lab type the following in the terminal:
+
+```
+jupyter lab
+```
+
+A browser should have launched and you should see a page that looks like the screenshot below. Now click on "R" notebook (circled in red on the screenshot below) to launch an Jupyter Lab with an R kernel.
+
+![](/resources_pages/imgs/jupyter_lab_r_kernel.png)
+
+Sometimes a kernel loads, but doesn't work as expected. To test whether your installation was done correctly now type `library(tidyverse)` in the code cell and click on the run button to run the cell. If your R kernel works you should see something like the image below:
+
+![](/resources_pages/imgs/jupyter_lab_r_kernel2.png)
 
 ## LaTeX
 
-Download the installer from [http://mirror.ctan.org/systems/texlive/tlnet/install-tl-windows.exe](http://mirror.ctan.org/systems/texlive/tlnet/install-tl-windows.exe). After downloading run the `install-tl-windows.exe` file and follow the installation instructions choosing the default options.
+We will install the lightest possible version of LaTeX and it's necessary packages as possible so that we can render Jupyter notebooks and R Markdown documents to html and PDF. If you have previously installed LaTeX, please uninstall it before proceeding with these instructions.
 
-*note - if a screen appears titled "Windows protected your PC", click on "More info" link and the "Run anyway" button*
+First, open RStudio and run the following commands to install the `tinytex` package and setup `tinytex`:
 
-After installation, restart Powershell and type the following to ask for the version:
+```
+install.packages('tinytex')
+tinytex::install_tinytex()
+```
+
+Note that you might see two error messages regarding lua during the installation, you can safely ignore these, the installation will complete successfully after clicking "OK".
```

Comment:
This is strange, literal error messages saying things failed, but it installs fine, also noted in the tinytex installation instructions so it will likely be widespread.